qiskit/algorithms/fidelities/__init__.py

```diff
@@ -0,0 +1,36 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
```

Comment:
By that I mean I expect to see that change as part of the PR where the init file is updated to include that link. This way the docs here will be built out - which I do not think they are at present since this is not linked in yet to the overall docs - adding a link to fidelities like is done with optimizers will link it in.

Reply:
Yes, sorry, I forgot push my latest changes last Friday. I had updated the `qiskit.algorithms` init file to include fidelities.